game_board/src/fr/lecampusnumerique/cda2025/javaalgo/boardgames/players/PlayerTurns.java

```diff
@@ -0,0 +1,8 @@
+package fr.lecampusnumerique.cda2025.javaalgo.boardgames.players;
+import fr.lecampusnumerique.cda2025.javaalgo.boardgames.games.Game;
+import java.util.Scanner;
+
+public class PlayerTurns  {
+    void switchPlayer(){
+    }
+}
```

Comment:
player turns should be a method withing the player class, no? isn't it a behavior that we expect from the player object? :)